tests/testthat/test-search_ranges.R

```diff
@@ -1,7 +1,15 @@
 context("Search ranges")
 
+check_version <- function() {
+  state<-hasRangeProcSupport()
+  if (!state) {
+    skip("Xapian Range Processor support not available")
+  }
+}
+
 test_that("search results are as expected", {
   
+  check_version()
```

Comment:
yes, will remove that. :)

On Wed, Aug 10, 2016 at 6:30 PM, James Aylett <notifications@github.com>
wrote:

> In tests/testthat/test-search_ranges.R
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74291092>:
>
> >  test_that("search results are as expected", {
> >
> > +  check_version()
>
> I think this is no longer needed, because of the latest work you've done
> :-)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/a79687e7e6e354e88aa771a7a9d7a60c92befba9#r74291092>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNoJ6-OUS1sH10gh0M3m3Q1DjgXyzks5qegrTgaJpZM4Jf4Xo>
> .
>